models/public/forward-tacotron/description/forward-tacotron.md

```diff
@@ -0,0 +1,132 @@
+# forward-tacotron
+
+## Use Case and High-Level Description
+
+ForwardTacotron is a model for the text-to-speech task originally trained in PyTorch\*
+then converted to ONNX\* format. ForwardTacotron performs mel-spectrogram regression from text.
+For details see [paper](https://arxiv.org/pdf/1703.10135.pdf), [paper](https://arxiv.org/pdf/1905.09263.pdf), [repository](https://github.com/as-ideas/ForwardTacotron).
+
+## ONNX Models
+We provide pretrained models in ONNX format for user convenience.
+
+### Steps to Reproduce training in PyTorch and Conversion to ONNX
+Model is provided in ONNX format, which was obtained by the following steps.
+
+1. Clone the original repository
+```sh
+git clone https://github.com/as-ideas/ForwardTacotron
+cd ForwardTacotron
+```
+2. Checkout the commit that the conversion was tested on:
+```sh
+git checkout 78789c1aa845057bb2f799e702b1be76bf7defd0
+```
+3. Follow README.md and train ForwardTacotron model.
+4. Copy provided script `forward_to_onnx.py` to ForwardTacotron root directory.
+5. Run provided script for conversion ForwardTacotron to onnx format
+```sh
+python3 forward_to_onnx.py --tts_weights checkpoints/ljspeech_tts.forward/fast_speech_step<iteration>K_weights.pyt
+```
+Notes:
+   1. Since ONNX doesn't support the build_index operation from pytorch pipeline, the model is divided into two parts: `forward_tacotron_duration_prediction.onnx, forward_tacotron_regression.onnx`.
```

Comment:
```suggestion
   1. Since ONNX doesn't support the build_index operation from PyTorch pipeline, the model is divided into two parts: `forward_tacotron_duration_prediction.onnx, forward_tacotron_regression.onnx`.
```